modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -89,7 +90,7 @@ Based on your role and the needs of your organization, you might perform the fol
 // | Master Organization Administrator | Change the default IdP across all environments and business groups. | Access Management | Resetting the default IdP updates all environments across all business groups to a selected IDP, without affecting existing client applications. This way, when a new IdP is created for the organization, all environments are updated with just one change.
 | Review to which client provider an application belongs on the client administrator page to be able to differentiate between different applications with the same name. | API Manager Administrator| Access Management | None
 | Configure an API in API Manager to automatically map to a default IdP configured by the business group administrator. The owner can also select a different IdP, if available. | API Owner | API Manager | None
-| When promoting or importing an API between environments, specify an IdP from the target environment. | API Owner | API Manager | If there is just one (default) IdP specified for the target environment, it is automatically assigned to the API on promotion. 
+| When promoting or importing an API between environments, specify an IdP from the target environment. | API Owner | API Manager | If there is just one (default) IdP specified for the target environment, it is automatically assigned to the API. 
```

Comment:
"Promoting" and "importing" don't really work with "between". Do we really mean "from one environment to another"? Also... "promoting"?

Reply:
API owner, API manager
lowercase

Reply:
See if you can omit the "Notes" column, which is a vague distinction.
In the case of this table row, the Note could be part of the task statement: When importing, if the target has only one, it is assigned to the API; otherwise, specify which target IdP to use.

Reply:
Hi Kim, the "Notes" column mostly functions as such. I think the example you describe is good, but the rest of the column is generally more notes-like.

The other two suggestions are fixed. Thanks!